src/plugins/github/README.md

```diff
@@ -0,0 +1,212 @@
+see the kinds of nodes and edges in that graph. The data is temporarily cached
+locally, but the source of truth for the GitHub plugin is always GitHub's live
+servers. That means that if content is deleted on GitHub, it will also
+disappear from the GitHub plugin (after a cache refresh).
+
+## Status and Caveats
+
+The GitHub plugin is still in beta. It tends to assign reasonable cred scores
+to contributors, based on their activity levels. However, it has difficulty
+assessing the value of different pull requests. A minor pull request that
+generated lots of review discussion or bikeshedding is likely to get a higher
+score than highly important work that merged easily.
+
+In general, the plugin mints cred based on _raw activity_. Depending on weight
+choices, every pull will mint a fixed amount of cred, likewise for every
+comment, issue, review, and so forth.
+
+This is problematic because, to put it simply, not all activity is equally
+valuable. Minting cred directly on raw activity encourages contributors to
+focus on quantity over quality. As an example, a poorly thought out pull
+request (which requires lots of review and feedback to fix) will generate more
+cred than a clean, elegant pull request (which merges with a minimum of fuss).
+
+We intend to improve the cred robustness of the GitHub plugin by
+adding better heuristics for assigning cred, e.g.:
+- minting cred when pulls are merged, rather than when they are created
+- allowing custom labels that influence the cred minted to pulls
+- modifying cred minting based on metrics like the size of the pull request
+
+We also intend to move cred minting away from raw activity and towards
+actions that require review, e.g.:
+- minting cred when a pull request merges
+- minting cred when a release is created
+- minting cred when a feature is added (via the Initiatives Plugin)
+
+[GitHub]: https://github.com/
+
+## Nodes
+
+The GitHub plugin creates the following kinds of nodes:
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. The repo node has
+no timestamp, so setting a weight on the repository will have no effect (i.e.
+they do not mint cred). This will change when we switch to [CredRank].
+
+[CredRank]: https://github.com/sourcecred/sourcecred/issues/1686
+
+[sourcecred/sourcecred]: https://github.com/sourcecred/sourcecred
+
+### Issue
+
+A GitHub issue, e.g. [sourcecred/sourcecred#40]. Issues are connected to their
+author(s), to entities that they reference, to their comments, and to the
+containing GitHub repository.
+
+[sourcecred/sourcecred#40]: https://github.com/sourcecred/sourcecred/issues/40
+
+### Pull Request
+
+A GitHub pull request, e.g. [sourcecred/sourcecred#35][pull]. Pulls are
+connected to their author(s), to entities they reference, to their comments,
+their reviews, to their containing repository, and, (if merged) to the commit
+that they merged as.
```

Comment:
>merged as

merged in?

Reply:
Updated to "the commit that they created when merged".